core-api/src/test/java/com/optimizely/ab/OptimizelyTest.java

```diff
@@ -3437,105 +2361,53 @@ public void activateWithListener() throws Exception {
             testUserAttributes.put("browser_type", "chrome");
         }
 
-        testUserAttributes.put(testBucketingIdKey, testBucketingId);
-
-        ActivateNotificationListener activateNotification = new ActivateNotificationListener() {
-            @Override
-            public void onActivate(@Nonnull Experiment experiment, @Nonnull String userId, @Nonnull Map<String, ?> attributes, @Nonnull Variation variation, @Nonnull LogEvent event) {
-                assertEquals(experiment.getKey(), activatedExperiment.getKey());
-                assertEquals(bucketedVariation.getKey(), variation.getKey());
-                assertEquals(userId, testUserId);
-                for (Map.Entry<String, ?> entry : attributes.entrySet()) {
-                    assertEquals(testUserAttributes.get(entry.getKey()), entry.getValue());
-                }
-
-                assertEquals(event.getRequestMethod(), RequestMethod.GET);
```

Comment:
Was this a bug?

Reply:
The mocked `logEventToDispatch` was a GET, removing the mock and using the actual `EventFactory` meant this needed to change to POST.